### ISSUE TITLE
New rule: require-multiline-attribute-newline

### ISSUE THREAD

**Issue by OJFord** [ota-meshi/eslint-plugin-svelte#362]
### Motivation

More readable multiline attribute blocks.

### Description

When an attribute gets long, particularly spanning multiple lines, IMO it's not very readable to have them (start) on the same line as the prop or directive.

### Examples

```svelte
<!-- ✓ GOOD -->
<Foo
  on:click={
    () => {
      foo = bar
    }
  }
/>

<!-- ✗ BAD -->
<Foo
  on:click={() => {
    foo = bar
  }}
/>
```

**Comment by ota-meshi** [comment]
Thanks for the rule suggestions.
That's an interesting rule. The new rule may conflict with the existing `mustache-spacing` rule, so I think the check by the `mustache-spacing` should be loosened.
https://ota-meshi.github.io/eslint-plugin-svelte/rules/mustache-spacing/

Also, I think the new rule should allow some options to disallow newlines. So I think it's a good idea to name the new rule something like `mustache-newline`.

**Comment by OJFord** [comment]
I think your naming suggestion covers it too - but just to add it occurred to me that this is just as relevant for templated text content:
```svelte
<!--  ✓ GOOD -->
<Foo>
  {
    generateSomething({
      param1: 123,
      param2: 456,
    })
  }
</Foo>

<!-- ✗ BAD -->
<Foo>
  {generateSomething({
      param1: 123,
      param2: 456,
    })}
</Foo>
 ```
